### ISSUE TITLE
Doesn't copy to [appname].css

### ISSUE THREAD

**Issue by richrout** [quaertym/ember-cli-compass-compiler#72]
Hi,

I can't get ember-cli-compass-compiler to compile to [appname].css as per the docs.
Ember CLI version 1.13.14. Reproduce:

```
ember new bla
cd bla
ember install ember-cli-compass-compiler
-- rename /app/styles/app.css to app.scss and add some content --
ember s
```

app.css is created (and in dist folder), but no bla.css

**Comment by quaertym** [comment]
What is your `ember-cli-compass-compiler` version?

**Comment by richrout** [comment]
I cleared my npm cache, it's version `0.3.2`
Probably worth noting my compass version is `1.0.3`

**Comment by repl-andrew-ovens** [comment]
Bump

**Comment by gskaplan** [comment]
Is there any status on this? I have this exact same problem. It generates an app.css file, not an appname.css file.

**Comment by quaertym** [comment]
@gskaplan Have you tried using the latest version?

**Comment by gskaplan** [comment]
I believe I am using the latest - this only started happening when I switched from 0.3.1 to 0.4.0. Here's what I'm using:

ember: 2.4.1
ember-cli:  2.4.2
compass 1.0.3
ember-cli-compass-compiler: 0.4.0

Here's the package.json file just for fun:

```javascript
"devDependencies":  {
    "body-parser": "^1.15.0",
    "broccoli-asset-rev": "^2.2.0",
    "ember-cli": "2.4.2",
    "ember-cli-app-version": "^1.0.0",
    "ember-cli-babel": "^5.1.5",
    "ember-cli-compass-compiler": "0.4.0",
    "ember-cli-content-security-policy": "0.5.0",
    "ember-cli-dependency-checker": "^1.2.0",
    "ember-cli-htmlbars": "^1.0.1",
    "ember-cli-htmlbars-inline-precompile": "^0.3.1",
    "ember-cli-ic-ajax": "0.2.4",
    "ember-cli-inject-live-reload": "^1.3.1",
    "ember-cli-moment-shim": "1.1.0",
    "ember-cli-qunit": "^1.2.1",
    "ember-cli-release": "0.2.8",
    "ember-cli-sri": "^2.1.0",
    "ember-cli-uglify": "^1.2.0",
    "ember-cli-windows-addon": "^1.3.1",
    "ember-data": "2.4.0",
    "ember-disable-proxy-controllers": "^1.0.1",
    "ember-export-application-global": "^1.0.4",
    "ember-load-initializers": "^0.5.0",
    "ember-moment": "6.0.0",
    "ember-resolver": "^2.0.3",
    "ember-truth-helpers": "1.2.0",
    "express": "^4.13.4",
    "glob": "^7.0.3",
    "loader.js": "^4.0.0",
    "morgan": "^1.7.0"
  },
  "dependencies": {
    "broccoli-funnel": "^1.0.1"
  }
```

**Comment by repl-andrew-ovens** [comment]
This is only an issue on Windows. Please merge my pull request and cut a
new release

**Comment by gskaplan** [comment]
I don't know how to do that since it is part of my build and I'm using npm to fetch it. Is there a fix planned for the near future? Or can you tell me if there is an easy way to merge and pull your request into my project?

**Comment by repl-andrew-ovens** [comment]
Sorry, that was meant for the owner. In the meantime, you can change your
package.json to use this instead of a version number:
repl-andrew-ovens/ember-cli-compass-compiler#df6a5647a0e28dad3823a439784fa14deaba6e80

**Comment by quaertym** [comment]
@repl-andrew-ovens Can you rebase so that it merges cleanly?

**Comment by repl-andrew-ovens** [comment]
@quaertym Done :)

**Comment by gskaplan** [comment]
Is there an ETA when this will be released? I will continue to use 0.3.1 until then. Thanks!
